server/src/test/java/org/opensearch/index/translog/transfer/BaseTranslogTransferManagerTests.java

```diff
@@ -453,29 +466,66 @@ public void testDownloadTranslog() throws IOException {
         assertTlogCkpDownloadStats();
     }
 
+    public void mockResponseDownloadBlobWithMetadata_WithNULLMetadata() throws IOException {
+        when(transferService.downloadBlobWithMetadata(any(BlobPath.class), eq("translog-23.tlog"))).thenAnswer(invocation -> {
+            Thread.sleep(delayForBlobDownload);
```

Comment:
why is there a sleep required and that too for 1ms?

Reply:
This is required to test value of time `getTotalDownloadTimeInMillis` in remoteTranslogTransferTracker.